EzMap.Api/Controllers/PoiCollectionController.cs

```diff
@@ -18,10 +18,12 @@ public class PoiCollectionController : ControllerBase
         [FromServices] IUnitOfWork uow,
         [FromServices] IIdentityService identityService)
     {
-        if (string.IsNullOrEmpty(dto.Name)
-            && string.IsNullOrEmpty(dto.Description))
+        var validator = new PoiCollectionCreateDtoValidator();
```

Comment:
This is okay but not optimal, it's still the old boilerplate-y code (Also the error message should comes from the validator as well). 
We can configure asp.net to use these validators on input mapping thus eliminate the need for these manual validator creating + invoking, see https://docs.fluentvalidation.net/en/latest/aspnet.html#automatic-validation, specifically https://docs.fluentvalidation.net/en/latest/aspnet.html#using-a-filter

Reply:
apply automatic valition from fluent validation